src/Controller/ViewController.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\View;
+use App\Repository\ViewRepository;
+use App\Service\ViewHelperService;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Exception\SessionNotFoundException;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\Routing\Annotation\Route;
+
+#[Route('/admin/view')]
+class ViewController extends AbstractController
+{
+    private const SESSION_KEY = self::class;
+    private const CREATEFORM_LAST_STEP = 3;
+
+    #[Route(['', '/list'], name: 'app_view_list')]
+    public function list(Request $request, ViewRepository $viewRepository): Response
+    {
+        return $this->render('view/list.html.twig', [
+            'views' => $viewRepository->findAll(),
+        ]);
+    }
+
+    #[Route('/add', name: 'app_view_add')]
+    public function add(Request $request, ViewRepository $viewRepository, ViewHelperService $viewHelperService): Response
+    {
+        // Create session for multistep form.
+        try {
+            $session = $request->getSession();
```

Comment:
What happens if the user leaves the form half filled and comes back to it after a week?

Reply:
I think we need a way of resetting the form session.